app/src/main/java/com/keylesspalace/tusky/components/domainblocks/DomainBlocksAdapter.kt

```diff
@@ -0,0 +1,27 @@
+package com.keylesspalace.tusky.components.domainblocks
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.paging.PagingDataAdapter
+import com.keylesspalace.tusky.components.followedtags.FollowedTagsAdapter.Companion.STRING_COMPARATOR
+import com.keylesspalace.tusky.databinding.ItemMutedDomainBinding
+import com.keylesspalace.tusky.util.BindingHolder
+
+class DomainBlocksAdapter(
+    private val onUnmute: (String) -> Unit
+) : PagingDataAdapter<String, BindingHolder<ItemMutedDomainBinding>>(STRING_COMPARATOR) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): BindingHolder<ItemMutedDomainBinding> {
+        val binding = ItemMutedDomainBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return BindingHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: BindingHolder<ItemMutedDomainBinding>, position: Int) {
+        getItem(position)?.let { instance ->
+            holder.binding.mutedDomain.text = instance
+            holder.binding.mutedDomainUnmute.setOnClickListener {
+                onUnmute(instance)
+            }
+        }
```

Comment:
```suggestion
        getItem(position)?.let { domain ->
            holder.binding.mutedDomain.text = domain
            holder.binding.mutedDomainUnmute.setOnClickListener {
                onUnmute(domain)
            }
        }
```

Couldn't figure out why this used both "instance" (which apparently is deprecated in favour of "server") and "domain".